content/docs/guides/laravel.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Connect from Laravel to Neon
+enableTableOfContents: true
+redirectFrom:
```

Comment:
Thank you for the contribution, @octoper. 
Since this is a new file, no need for redirects from a preexisting file or location.  You can remove these lines.
redirectFrom:
  - /docs/quickstart/laravel
  - /docs/integrations/laravel
 Otherwise, it looks good to me!

Reply:
Done @danieltprice !